src/components/Button/styles.module.css

```diff
@@ -0,0 +1,44 @@
+.btn {
+  border-radius: 9999px;
+}
+
+.solid {
+  color: #fff;
+  background-image: linear-gradient(to right, var(--politico-blue), var(--politico-dark-blue));
+}
+
+.solid:hover {
+  filter: brightness(1.15);
+  box-shadow: var(--shadow-md);
+  transform: translateY(-1px);
+}
+
+.solid:active {
+  filter: brightness(0.9);
+  box-shadow: none;
+  transform: none;
+}
+
+.outline {
+  font-weight: 500;
+  color: var(--slate-800);
+  border-width: 1px;
+  border-color: var(--slate-800);
+}
+
+.outline:hover {
+  color: var(--slate-100);
+  background-color: var(--slate-800);
+}
+
+.outline:active {
+  filter: brightness(1.33);
+}
+
+.ghost {
+  color: #334155;
```

Comment:
In elections we kept all colors together as variables. I think it's worth keeping that paradigm around, it made simplifying colors much easier.